docs/conf/inspircd.conf.example

```diff
@@ -864,14 +864,10 @@
 
 <badnick
          # nick: Nick to disallow. Wildcards are supported.
-         nick="ChanServ"
+         nick="Kyubey"
```

Comment:
Can you change this back to what it was? I understand that it'll be a duplicate because ChanServ is in generic.conf.example but duplicates are silently ignored and that way there's no chance that a perfectly okay nick will get blocked due to people being lazy to edit this.